pkgs/development/python-modules/jc/default.nix

```diff
@@ -4,20 +4,18 @@
 , ruamel_yaml
 , xmltodict
 , pygments
-, isPy27
 , pytestCheckHook
 }:
 
 buildPythonPackage rec {
   pname = "jc";
-  version = "1.15.4";
-  disabled = isPy27;
```

Comment:
Min. version is [Python 3.6](https://github.com/kellyjonbrazil/jc/blob/master/setup.py#L20).